packages/atlas/src/utils/popperModifiers.ts

```diff
@@ -0,0 +1,37 @@
+import { ModifierArguments, ModifierPhases } from '@popperjs/core'
```

Comment:
Correct me if I'm wrong but we don't need to install another package just to get the modifier types. 
We can use `Modifier` from 'react-popper. For example: 
```
export const popperIndexModifier: Modifier<'popperIndex'> = {
  name: 'popperIndex',
  enabled: true,
  phase: 'beforeWrite',
  requires: ['computeStyles'],
  fn: ({ state }) => {
    state.styles.popper.zIndex = String(zIndex.snackbars)
  },
  effect: ({ state }) => {
    state.elements.popper.style.zIndex = String(zIndex.snackbars)
  },
}
```